tools/autograd/derivatives.yaml

```diff
@@ -693,63 +721,81 @@
 - name: lerp.Scalar(Tensor self, Tensor end, Scalar weight) -> Tensor
   self: "weight.isComplex() ? grad * (1 - weight.conj().toComplexDouble()) : grad * (1 - weight.toDouble())"
   end: grad * weight.conj()
+  result: at::lerp(self_t, end_t, weight)
 
 - name: lerp.Tensor(Tensor self, Tensor end, Tensor weight) -> Tensor
   self: grad * (1 - weight).conj()
   end: grad * weight.conj()
   weight: grad * (end - self).conj()
+  result: at::lerp(self_t, end_t, weight_p) + weight_t * (end_p - self_p)
 
 - name: lgamma(Tensor self) -> Tensor
   self: grad * digamma(self)
+  result: auto_element_wise
 
 - name: digamma(Tensor self) -> Tensor
   self: grad * polygamma(1, self)
+  result: auto_element_wise
 
 - name: polygamma(int n, Tensor self) -> Tensor
   self: grad * polygamma(n + 1, self)
+  result: auto_element_wise
 
 - name: polygamma_(Tensor(a!) self, int n) -> Tensor(a!)
   self: grad * polygamma(n + 1, self)
+  result: auto_element_wise
 
 - name: log(Tensor self) -> Tensor
   self: grad.div(self.conj())
+  result: auto_element_wise
 
 - name: log10(Tensor self) -> Tensor
   self: grad / (self.conj() * 2.3025850929940456)
+  result: auto_element_wise
 
 - name: log1p(Tensor self) -> Tensor
   self: log1p_backward(grad, self)
+  result: auto_element_wise
 
 - name: log2(Tensor self) -> Tensor
   self: grad / (self.conj() * 0.6931471805599453)
+  result: auto_element_wise
 
 - name: logaddexp(Tensor self, Tensor other) -> Tensor
   self: grad / (1 + exp(other - self))
   other: grad / (1 + exp(self - other))
+  result: self_t / (1 + exp(other_p - self_p)) + other_t / (1 + exp(self_p - other_p))
 
 - name: logaddexp2(Tensor self, Tensor other) -> Tensor
   self: grad / (1 + pow(2, other - self))
   other: grad / (1 + pow(2, self - other))
+  result: self_t / (1 + pow(2, other_p - self_p)) + other_t / (1 + pow(2, self_p - other_p))
 
 - name: xlogy.Tensor(Tensor self, Tensor other) -> Tensor
   self: grad * at::xlogy((self != 0), other)
   other: grad * self / other
+  result: self_t * at::xlogy((self_p != 0), other_p) + other_t * self_p / other_p
```

Comment:
(no action required) Lots of binary elementwise formulas here that we could have codegen'ed with a special "auto_element_wise"